cluster/gce/util.sh

```diff
@@ -662,6 +662,27 @@ EOF
 fi
 }
 
+# copy controller config into a temporary file.
```

Comment:
per discussion before, please don't use file generation.

Reply:
After discussion with Sonya, she suggests to use this method, so we decide to use this method yo get working.

Reply:
we can discuss in team meeting again to get clarification.

Reply:
get clarification in the meeting.